clouddrift/select.py

```diff
@@ -1,40 +1,41 @@
-import awkward as ak
+import xarray as xr
 import numpy as np
+import warnings
 
 
-def mask_var(var: ak.Array, value):
+def mask_var(var: xr.DataArray, value):
     """Return the mask of a subset of the data matching a test criterion.
 
     Args:
-        var (ak.Array): ak.Array
+        var (xr.DataArray): ak.Array
         value: tuple, list or scalar defining a test criterion
 
     Returns:
-        mask (Ak.Array): where values corresponding to the criterion are True
+        mask (xr.DataArray): where values corresponding to the criterion are True
     """
 
     if isinstance(value, tuple):  # min/max defining range
         mask = np.logical_and(var >= min(value), var <= max(value))
```

Comment:
If we use `tuple` to mean expressing a range of values, I think we should constrain to a 2-tuple, and use `(min, max)` explicitly. I.e.:

```suggestion
        mask = np.logical_and(var >= value[0], var <= value[1])
```

Rationale:

1. If we let tuples have any number of values, it becomes easier for the user to confuse the `tuple` and the `list` and get two completely different behaviors.
2. There is a use-case in data analysis and modeling where a `(min, max)` pattern with `min > max` is used to return an empty subset. This is not possible with the current implementation.